evergreen/evergreen.yml

```diff
@@ -214,6 +214,8 @@ functions:
       params:
         working_dir: mongo-csharp-driver
         script: |
+          export FLE_AWS_ACCESS_KEY_ID=${FLE_AWS_ACCESS_KEY_ID}
```

Comment:
Why export instead of:

    FLE_AWS_ACCESS_KEY_ID=${FLE_AWS_ACCESS_KEY_ID} FLE_AWS_SECRET_ACCESS_KEY=${FLE_AWS_SECRET_ACCESS_KEY} ... evergreen/run-tests.sh?\

? Is it because run-tests.sh echos them if you do it that way?

Reply:
Yep, you're correct! More specifically, it's because `${PREPARE_SHELL}` results in `set -o xtrace` which results in all commands being echoed as they're executed.